### DISCUSSION TITLE
Syncfusion blazor example

### DISCUSSION THREAD

**Discussion by markagama** [oqtane/oqtane.framework#3062] [Q&A]
Great example provide by Michael on how to add Syncfusion to Oqtane. I was wondering, @ADefWebserver if you're going to update the example to be compatible with Oqtane 4. Any hints on what changes are needed for the Oqtane 4 environment would be helpful.

**Comment by ADefWebserver** [comment]
@markagama Due to my schedule I have no plans to update that example. However, it should work with Oqtane 4.

**Reply by markagama** [reply]
thx for the response @ADefWebserver - I appreciate the confirmation that it should work in Oqtane 4.

A side note, you have two methods to add the license to the server startup file. earlier in the example you suggest to hard code and later to use the code when it is placed in the json file. I assume it is likely to be more secure? to have the license code in the json file?

**Comment by ADefWebserver** [comment]
@markagama The only difference is that it is easier to update if it is in the .json file (because you don't have to re-compile the application). The security risk is the same.

**Comment by markagama** [comment]
I followed the steps provided by @ADefWebserver but was unable to get the syncfusion blazor grid control to work with the new Oqtane release. Has anyone else tried this? Or has anyone seen an example of how to get Syncfusion controls working with the latest Oqtane?
